From 3f3c28f2f7ec3f2c06ca889dae488df26989d293 Mon Sep 17 00:00:00 2001
From: "Cheng.Yenmin" <Cheng.Yenmin@inventec.com>
Date: Fri, 6 Nov 2020 08:15:11 +0800
Subject: [PATCH linux dev-4.10] Commit reason: Changed the IPMI SEL log
 storage path. Signed-off-by: yenmin <cheng.yenmin@inventec.com>
To: openbmc@lists.ozlabs.org
Cc: joel@jms.id.au

 Author:    Cheng.Yenmin <Cheng.Yenmin@inventec.com>
 Changes to be committed:
	modified:   include/sel_logger.hpp
---
 include/sel_logger.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/sel_logger.hpp b/include/sel_logger.hpp
index 0d8c38f..b28e7af 100644
--- a/include/sel_logger.hpp
+++ b/include/sel_logger.hpp
@@ -33,7 +33,7 @@ static constexpr uint16_t selInvalidRecID =
 static constexpr size_t selEvtDataMaxSize = 3;
 static constexpr size_t selOemDataMaxSize = 13;
 
-static const std::filesystem::path selLogDir = "/var/log";
+static const std::filesystem::path selLogDir = "/var/lib/ipmi";
 static const std::string selLogFilename = "ipmi_sel";
 
 template <typename... T>
-- 
2.25.1


